src/Core/Cache/DefaultCacheFactory.php

```diff
@@ -85,7 +85,8 @@ protected function isAPCUSupported()
     {
         static $apcuSupported = null;
         if (null === $apcuSupported) {
-            $apcuSupported = ApcuAdapter::isSupported();
+            // Need to check for CLI because Symfony won't: https://github.com/symfony/symfony/pull/25080
+            $apcuSupported = PHP_SAPI === 'cli' ? ini_get('apc.enable_cli') && function_exists('apcu_fetch') : ApcuAdapter::isSupported();
```

Comment:
Can you please use `Director::is_cli()`. You should probably always check `ApcuAdaptor::isSupported()` in either case.

Reply:
Had a look around and I can't see a way to enable for CLI-only (and also that would be super weird) so I'm happy to do that.

Wanted to use `Director::is_cli()` in the Symfony PR, grumbled, forgot to use it here 😅

Reply:
Also, is there a reason this is done is this roundabout manner? Why can't we simply return the ternary? Same with the `PHPFiles` check below